CHANGELOG.rst

```diff
@@ -9,6 +9,7 @@ Unreleased
 ==================
 * feat: Remove Add cta and hide delete dropdown actions from AliasContent admin ChangeList
 * feat: Site field added to plugin
+* fix: Added parler to the installation instructions as an installed app
```

Comment:
```suggestion
```

Needs moving to the unreleased section, I think the fork you had was not updated.